Gruntfile.js

```diff
@@ -59,6 +60,13 @@ module.exports = function (grunt) {
 
     // The actual grunt server settings
     connect: {
+      proxies: [
+        {
+           context: '/es_proxy',
+           host: 'localhost',
+           port: 9001
```

Comment:
I don't think you'll end up needing the port 9001 proxy at all - everything you want should be configurable on grunt-connect-proxy:

```suggestion
           context: '/es_proxy',
           host: ES_HOST,
           port: 9200, 
           rewrite: {
             '^/es_proxy': ''
           }
```

(https://www.npmjs.com/package/grunt-connect-proxy#optionsrewrite)

Reply:
I was considering that. I wonder if we need to keep the back-end server to add 'rules' down the road. For example, if we want to block certain endpoints such as _bulk or _create? Is that valid concern? Or should we cross that bridge when we get to it?

Reply:
You can block everything that's not a GET in middleware.

Reply:
Removed the back-end and doing the proxy-ing in Grunt